src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientOptionalParametersTest.java

```diff
@@ -139,6 +154,14 @@ private void assertStreamOptions(RecordedRequest request) throws UnsupportedEnco
       assertThat(request.getPath())
           .contains("delete_after_seconds=" + sessionConfig.getDeleteAfterSeconds());
     }
+    if (sessionConfig.getStartTs() != null) {
+      assertThat(request.getPath())
+              .contains("start_ts=" + sessionConfig.getStartTs());
```

Comment:
Don't think this needs two lines
```suggestion
      assertThat(request.getPath()).contains("start_ts=" + sessionConfig.getStartTs());
```
But if two lines is wanted, will need proper indentation
```suggestion
      assertThat(request.getPath())
        .contains("start_ts=" + sessionConfig.getStartTs());
```